<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductDetailPage</class>
 <widget class="QWidget" name="ProductDetailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="mainLayout" stretch="1,1">
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>16</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>16</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="leftVLayout" stretch="0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="sellerLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="addDateLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightVLayout" stretch="1,1,0,0,1">
       <item>
        <widget class="QLabel" name="productNameLabel">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="priceLayout" stretch="1,0">
         <item>
          <widget class="QLabel" name="productPriceLabel">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: green;</string>
           </property>
           <property name="text">
            <string>Cena jednostkowa</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="productPriceNumLabel">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="amountLabel">
         <property name="text">
          <string>Liczba sztuk</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="amountLayout" stretch="0,0,9">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="amountSpinBox">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
           </property>
           <property name="specialValueText">
            <string>1</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="availableAmountLabel">
           <property name="styleSheet">
            <string notr="true">color: grey;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonsLayout">
         <item>
          <widget class="QPushButton" name="deleteProductButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Usu≈Ñ produkt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addToCartButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Dodaj do koszyka</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
